machine/quanta/quanta_bwde/rootconf/sysroot-lib-onie/init-platform

```diff
@@ -0,0 +1,33 @@
+# -*- shell-script -*-
+
+#  Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>
+#  Copyright (C) 2017 Audi Hsu <audi.hsu@quantatw.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+# This script fragment is sourced by /etc/init.d/init-arch.sh, which
+# in turn executes the init_platform_pre_arch() and
+# init_platform_post_arch() functions.
+
+# A machine can use this feature to run any early boot scripts needed
+# by the machine.  A typical usage would be to initialize specific
+# hardware devices.
+
+# Use this function to perform any initializations required by the
+# architecture specific initialization.  This function executes before
+# the architecture initializations.
+init_platform_pre_arch()
+{
+    echo "Running init_platform_pre_arch..."
+    echo "Mount devtmpfs..."
+    mount -t devtmpfs devtmpfs /dev
+    mkdir /dev/pts
+    mount -t devpts devpts /dev/pts
```

Comment:
Actually, we found that CONFIG_DEVTMPFS_MOUNT does not work with cpio.xz format. It just works with initramfs in previous kernel version. Therefore, the common way to make this work is to add ``mount -t devtmpfs none /dev`` in the very first part of system init, such as inittab or init script.